src/main/java/igrad/logic/parser/CourseBookParser.java

```diff
@@ -56,20 +52,14 @@ public Command parseCommand(String userInput) throws ParseException {
         case ClearCommand.COMMAND_WORD:
             return new ClearCommand();
 
-        case FindCommand.COMMAND_WORD:
-            return new FindCommandParser().parse(arguments);
-
-        case ListCommand.COMMAND_WORD:
-            return new ListCommand();
-
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
 
         default:
-            throw new ParseException(MESSAGE_UNKNOWN_COMMAND);
+            throw new ParseException(Messages.MESSAGE_UNKNOWN_COMMAND);
```

Comment:
oh i see why, you used `Messages.MESSAGE_UNKNOWN_COMMAND`, instead of static import (please refer to comment (directly above this))

guess I'll just change it back to static import

Reply:
Yeah sorry its supposed to be static import.
Some of them changed to non-static when I refactored, I tried my best to fix to static but I guess I missed some, my bad!